www/install/steps/process/createDbUser.php

```diff
@@ -97,11 +97,13 @@
 
     // checking mysql version before trying to alter the password plugin
     $prepareCheckVersion = $link->query($checkMysqlVersion);
+    $versionName = false;
```

Comment:
I don't really know... but may be better to initialize to null instead of false

Reply:
false and null will have the same effect on next condition :
https://github.com/centreon/centreon/pull/8166/files#diff-fc4156f2d8d42892a099d285271ce969R109

Reply:
Kevin's right.
Be consistent, a variable that should receive a string should not be initialized with a boolean!

Reply:
is null a valid string ? how should I initialize it ?
if no value is returned from the query, the variable won't be set